src/three-components/Renderer.js

```diff
@@ -60,7 +63,8 @@ export default class Renderer extends EventDispatcher {
       canvas: this.canvas,
       context: this.context,
     });
-    this.renderer.autoClear = false;
+    // TODO: hook to transparent attribute
+    this.renderer.autoClear = isTransparent;
```

Comment:
A handful of components use the renderer outside of elements -- composing environment maps or shadows, or any WebGLRenderTargets -- `autoClear` should not change as they affect all of those framebuffers

Reply:
How can I keep it `false` and avoid other scenes bleeding in?

Reply:
@forresto we do per-scene (in other words, per-element) renderer configuration in the render loop itself. See https://github.com/GoogleWebComponents/model-viewer/blob/master/src/three-components/Renderer.js#L135-L171.

However, this may not be suitable for all kinds of renderer configuration, and `autoClear` is probably not going to be suitable as it clears the whole context.

In the future, we intend to support additional rendering modes (Jordan already referenced this, but I'll ping it again as an example: https://github.com/GoogleWebComponents/model-viewer/issues/216). As we start to get into that, we should be able to layer on configurations like this.

Reply:
Are you recommending that I hold more work on this PR for #216? I can use my branch for now, since it seems to work.

Reply:
👍 thanks @forresto yeah I think it would be best to pin this PR for now and come back to it soon as we start to upgrade our approaches to rendering.